rest-api/tools/import_organizations.py

```diff
@@ -181,15 +181,15 @@ def _entity_from_row(self, row):
 
   def _get_lat_long_for_site(self, address_1, city, state):
     try:
-        api_key = os.environ.get('API_KEY')
-        gmaps = googlemaps.Client(key=api_key)
-        geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
-        latitude = geocode_result['geometry']['location']['lat']
-        longitude = geocode_result['geometry']['location']['lng']
-        return latitude, longitude
+      api_key = os.environ.get('API_KEY')
+      gmaps = googlemaps.Client(key=api_key)
+      geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
```

Comment:
Based on the documentation, it sounds like if you enter in a bad address here, you'll get back no results. We need to handle that case gracefully, logging a message and returning None from this function... or perhaps raising an exception and bailing out, but in a way that makes it clear that the address is wrong.

We should also check for the presence of geometry, location, and lat/lng below before accessing them... it seems possible these won't always come back even if an entry is present in the results.